icons/arrow_upward.svg

```diff
@@ -0,0 +1 @@
+material/baseline-arrow_upward-24px.svg
```

Comment:
Why don't we just rename the file?

Reply:
To keep track that it's a material icon.